azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/mqtt/async_adapter.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from .mqtt_transport import MQTTTransport
+from azure.iot.common import asyncio_compat
+
+logger = logging.getLogger(__name__)
+
+
+class MQTTAsyncTransport(MQTTTransport):
+    """
+    Asynchronous implementation of MQTTTransport for communication via MQTT protocol.
+    """
+
+    async def connect(self, callback=None):
+        logger.info("async connecting to transport")
+        connect_async = asyncio_compat.emulate_async(super(MQTTAsyncTransport, self).connect)
```

Comment:
>super(MQTTAsyncTransport, self) [](start = 53, length = 31)

This module will only come to play when we are at python 3 right ? May be we can start using `super().methoName(args)` as python 3 supports this as opposed to `super(subClass, instance).method(args)`